node/core/provisioner/src/lib.rs

```diff
@@ -271,7 +271,15 @@ async fn send_inherent_data_bg<Context>(
 
 		match send_result.await {
 			Err(err) => {
-				gum::warn!(target: LOG_TARGET, err = ?err, "failed to assemble or send inherent data");
+				if let Error::CanceledBackedCandidates(_) = err {
```

Comment:
I think we can make it more readable by matching `Err(Error::CanceledBackedCandidates(_))` and `Err(err)` and getting rid of the nested if.

Reply:
Oh that's a good one. Thanks!

Reply:
Actually, I'd like to keep that. The nested if is just for logging, if we get more cases in the future, I would rather make it a function `log_error`.